package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "tsdoc-markdown",
-  "version": "0.0.4",
+  "version": "1.0.0",
```

Comment:
Can you please revert this change? I'll take care of releasing, most probably `v0.1.0`.

Reply:
Will do.